src/containers/oracle.js

```diff
@@ -201,16 +207,18 @@ class OraclePage extends React.Component {
         onApprove(oracle.topicAddress, ORACLE_BOT_THRESHOLD, senderAddress);
 
         setTimeout(() => {
+          console.log('onSetResult');
           onSetResult(contractAddress, selectedIndex, senderAddress);
         }, SUB_REQ_DELAY);
         break;
 
       case 'VOTING':
 
         /** The amount of voting needs to be approved by Bodhi_token * */
-        onApprove(oracle.topicAddress, amount, senderAddress);
+        onApprove(_.trimStart(oracle.address, '0x'), amount, senderAddress);
 
```

Comment:
Every address's "0x" needs to be trimmed before send to api. I am open to suggestions.

Reply:
Already fixed in API/qweb3 layer. No need to trim any addresses of any sort of the `0x`.